solutions/5_betty.js

```diff
@@ -0,0 +1,26 @@
+// Betty Chow
+
+const solution = (arr) => {
+  let newArr = [];
+  for(let j = 0; j < arr.length; j++) {
+    if (typeof arr[j] === 'string') {
+      arr[j].toLowerCase();
+    }
+  }
+  while( arr.length > 0) {
+    let smallestNum = arr[0];
+    for(let i=0; i < arr.length; i++) {
+      if (arr[i] < smallestNum) {
+          smallestNum = arr[i];
+      }
+    }
+    let x = arr.indexOf(smallestNum);
+      newArr.push(arr.splice(x , 1));
```

Comment:
Indentation.

And also, since you are always only removing one element, you can directly push the first element `newArr.push(arr.splice(x , 1)[0]);`

Then, you won't have to use a new var called `finalArr` or `concat.apply` function in line 20.